ci/jenkins/test.sh

```diff
@@ -253,17 +253,16 @@ function collect_windows_network_info_and_logs {
 function wait_for_antrea_windows_pods_ready {
     kubectl apply -f "${WORKDIR}/antrea.yml"
     if [[ "${PROXY_ALL}" == false ]]; then
-        kubectl apply -f "${WORKDIR}/kube-proxy-${WINDOWS_YAML_SUFFIX}.yml"
+        #kubectl apply -f "${WORKDIR}/kube-proxy-${WINDOWS_YAML_SUFFIX}.yml"
```

Comment:
Remove it?

Reply:
this is a temporary change to run windows CI tests. Will revert it back after review and before merge.